x-pack/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -72,12 +77,42 @@ export class CspPlugin
       ...plugins.fleet,
     });
 
+    plugins.fleet.fleetSetupCompleted().then(async () => {
+      const packageInfo = await plugins.fleet.packageService.asInternalUser.getInstallation(
+        CIS_KUBERNETES_PACKAGE_NAME
+      );
+
+      // If package is installed we want to make sure all needed assets are installed
+      if (packageInfo) {
+        this.initialize(core);
```

Comment:
be aware, there's a use case that the package is installed without adding it to agent-policy.
That means `packagePolicyPostCreate` event is not being called.

so there's a case that the assets would be installed only on next server start - which is fine